cpp/src/traversal/traversal_common.cuh

```diff
@@ -204,6 +218,19 @@ binsearch_maxle(const IndexType* vec, const IndexType val, IndexType low, IndexT
   }
 }
 
+template <typename T>
+__device__ static __forceinline__ T atomicAdd(T* addr, T val)
+{
+  return ::atomicAdd(addr, val);
+}
+
+template <>
+__device__ __forceinline__ long atomicAdd<long>(long* addr, long val)
+{
+  return ::atomicAdd(reinterpret_cast<unsigned long long*>(addr),
```

Comment:
Better use int64_t instead of long and better add
static_cast(sizeof(unsigned long long) == sizeof(int64_t), "");

Reply:
Done.